core/src/main/scala/dagr/core/cmdline/parsing/CommandLineParser.scala

```diff
@@ -281,6 +282,14 @@ private[cmdline] class CommandLineParser[T](val targetClass: Class[T]) extends L
     builder.toString
   }
 
+  /** add extra whitespace after newlines for pipeline descriptions*/
+  private def formatLongDescription(description: String): String = {
+    val desc = description.stripMargin
+    desc
+      .dropWhile(_ == '\n')
```

Comment:
Nevermind, I forgot that dropwhile just drops until the first non-matching is found.